src/polymer-language-server.ts

```diff
@@ -284,5 +294,18 @@ handleErrors<Result, Fallback>(
   }
 }
 
+function log(val: any) {
+  if (!connection) {

```

Comment:
If the language server is running in stdio mode we shouldn't log anything to it, as it's a data channel (and it wouldn't be super useful if we did).